test/664-raceway.py

```diff
@@ -0,0 +1,10 @@
+# https://www.openstreetmap.org/way/28825404
+assert_has_feature(
+    16, 10476, 25242, 'roads',
+    { 'id': 28825404, 'kind': 'minor_road', 'highway': 'raceway' })
```

Comment:
> Should we assert the sort_key value in the tests?

We could for now, knowing that it makes the tests more brittle.

IIUC, what we really care about here with sort keys is the relative positioning between certain kinds of features. In the future, I'd rather that we created generative tests here instead and had the relative feature positions be our properties/invariants.